src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule232specialescape/InputSpecialEscapeSequences.java

```diff
@@ -14,7 +14,7 @@ public String wrongEscapeSequences() {
     final String r2 = "\u0009"; // violation 'Consider using special escape sequence .*'
     final String r3 = "\u000csssdfsd"; // violation 'Consider using special escape sequence .*'
     final String r4 = "\u1111sdfsd\444";
-
+    // violation above 'Unicode escape(s) usage should be avoided.'
```

Comment:
do we update 2.3.2 or 2.3.3 ?

https://github.com/checkstyle/checkstyle/tree/master/src/it/java/com/google/checkstyle/test/chapter2filebasic/rule233nonascii

2.3.2 should be only https://checkstyle.org/google_style.html#a2.3.2 
>  IllegalTokenText 

violation that you add is from https://checkstyle.org/checks/misc/avoidescapedunicodecharacters.html#Violation_Messages  that is 2.3.3

just move `final String r4 = "\u1111sdfsd\444";` to https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/InputNonAsciiCharacters.java

Reply:
done

https://github.com/checkstyle/checkstyle/blob/5e0f28445960eec284d65c5d2e20e12cc0fe9d82/src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/InputNonAsciiCharacters.java#L18-L19

All `Unicode escape(s)` violations were same, so I added the only one violation